packages/ember-toucan-core/src/components/button.hbs

```diff
@@ -9,7 +9,7 @@
     {{yield to="loading"}}
     <span class="sr-only" data-loading>{{yield}}</span>
   {{else if @isDisabled}}
-    <span class="flex flex-grow items-center gap-x-2">
+    <span class="flex flex-grow items-center justify-centre gap-x-2">
```

Comment:
Should this be:

```suggestion
    <span class="flex flex-grow items-center justify-center gap-x-2">
```

looking at the [Tailwind docs](https://tailwindcss.com/docs/justify-content#center)

Reply:
oops

Reply:
This does make me wonder...we have [prettier for tailwind](https://github.com/CrowdStrike/ember-toucan-core/blob/1c1d185bdb49f11cf937cf2797d112aef5983e1b/docs-app/.prettierrc.js#L14) but not any [eslint for tailwind](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md)...should we care about this?

Asking as someone only somewhat familiar with the tailwind linting setup...